### ISSUE TITLE
Outdated frame_system Weights

### ISSUE THREAD

**Issue by NachoPal** [paritytech/substrate#10270]
I ran the frame_system benchmarks for Statemint, Statemine & Westmint and I found out they differ from the ones in this repository.

https://github.com/paritytech/cumulus/pull/762

`set_heap_pages` now have an extra read an write, coming probably from this commit https://github.com/paritytech/substrate/commit/598d74bc067ab4532ee008e339f48278d3bf568b#diff-68096a50d12854e07693a4828590517bb81fea37a9253640278ecdc5b93b6992R363
when `deposit_log` was added

**Comment by thiolliere** [comment]
yes the doc is a bit weird, maybe we should remove them, but otherwise here is the fix https://github.com/paritytech/substrate/pull/10293
